tests/conftest.py

```diff
@@ -145,7 +145,7 @@ def pytest_addoption(parser):
     parser.addoption(
         '--base-branch',
         type=str,
-        default='master',
+        default='',
```

Comment:
This needs to be fixed?

Reply:
Got it. For something that will be set to default value in the underlying function, it should be None as default.

Also, we should use last minor version as default. It is a more common path, or can we just enable both last minor version and last master?